### ISSUE TITLE
Timeline and Hide Elements interaction

### ISSUE THREAD

**Issue by allanaaa** [omeka/plugin-Timeline#8]
When https://github.com/zerocrates/HideElements is active and hiding a date field from public view, the timeline cannot select that field to use as item date field or item interval field. 

I'd like to be able to use one field specifically for the timeline but not show that field on item view pages.

**Comment by matthewjmckinley** [comment]
I don't think this can happen from the Timeline side. After testing and looking at the HideElements code, it looks like selected Hidden fields are only removed from select forms (such as in Timeline) when their `Hide On: Search` parameter is selected. Specifically this line says to return the unaltered element Options array if Override is selected OR if `Hide On: Search` is NOT selected:

(https://github.com/zerocrates/HideElements/blob/6649f20df59d6fa4cd0ef2056c99755db6524bdd/HideElementsPlugin.php#L145)

Could possibly add another condition there to leave the options array alone if A) Timeline plugin is installed and B) it's the Timeline form calling specifically, but since I don't know the details of how `filterElementsSelectOptions()` is called, I don't know if that level of granularity is possible and/or if it might leave the element options unaltered on other forms where we DO want them removed.
